### PR TITLE
e2e:chore - test for the show vulnerabilities and project path flags

### PR DESCRIPTION
**Pull request by lucasbrunozup**
**- What I did**
E2e tests for the `--show-vulnerabilities-type` and `--project-path` flags

**- How to verify it**
Run `make test-e2e`

**- Description for the changelog**
N/A

Signed-off-by: lucas.bruno <lucas.bruno@zup.com.br>

### CODE REVIEW

File: e2e/commands/start/start_test.go [hunk 1]
```diff
@@ -96,9 +99,16 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 		})
 	})
 
+	When("--project-path is passed", func() {
+		It("Checks if the project path property was set", func() {
+			Expect(session.Out.Contents()).To(ContainSubstring(`\"disable_docker\": true`))
```
Comment (iancardosozup) [inline e2e/commands/start/start_test.go:104]: substring should be about `\"project_path\"` and not `\"disable_docker\":`

File: e2e/commands/start/start_test.go [hunk 2]
```diff
@@ -96,9 +99,16 @@ var _ = Describe("running binary Horusec with start parameter", func() {
...
+		})
+	})
+
 	When("--disable-docker is passed", func() {
 		It("Checks if the disable docker was set as true", func() {
-			Expect(session.Out.Contents()).To(ContainSubstring(`\"disable_docker\": true`))
+			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`\"project_path\": \"%s\"`, testutil.NormalizePathToAssertInJSON(projectPath))))
+			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`Project sent to folder in location: [%s`, testutil.NormalizePathToAssert(projectPath))))
```
Comment (iancardosozup) [inline e2e/commands/start/start_test.go:111]: seems like you misplaced the tests, this should be on `When("--project-path is passed", func()` and the other in `When("--disable-docker is passed", func() {`

File: e2e/commands/start/start_test.go (hunk 1 above)
Comment (lucasbrunozup) [reply e2e/commands/start/start_test.go]: Something wrong is not right 🧐, Fixed !

File: e2e/commands/start/start_test.go (hunk 2 above)
Comment (lucasbrunozup) [reply e2e/commands/start/start_test.go:105]: Fixed!

**Review by iancardosozup — APPROVED** [review]

**Review by nathanmartinszup — APPROVED** [review]

**Comment by matheusalcantarazup** [comment]
Please, resolve the conflicts @lucasbrunozup

**Pull request merged**